shoop/campaigns/forms.py

```diff
@@ -180,11 +182,40 @@ def save(self, commit=True):
 
         return instance
 
+    def _save_effects(self, effect_key, instance):
+        for cls in get_provide_objects(effect_key):
+            if not self.cleaned_data[cls.identifier]:
+                continue
+            field_value = self.cleaned_data[cls.identifier]
+            effect = cls.objects.create()
+            if effect.model:
+                effect.values = effect.model.objects.filter(pk__in=ast.literal_eval(field_value))
```

Comment:
Hmm.. `ast.literal_eval` doesn't seem like a right tool here. Can't the field_value be just a list of ids? I.e. as comma separated string, so then `pks = map(int, field_value.split(','))` would do.